File: RetroGameGauntlet.Forms/ViewModels/WikipediaItemViewModel.cs
using System;
namespace RetroGameGauntlet.Forms.ViewModels
{
    public class WikipediaItemViewModel : BaseViewModel
    {
        public string Title { get; set; }

        public string Description { get; set; }

        public string Url { get { return "https://en.wikipedia.org/wiki/" + Title; } }

        public Uri Uri { get { return new Uri(Url); } }
    }
}

File: RetroGameGauntlet.Core/ViewModels/Items/WikipediaItemViewModel.fs
namespace RetroGameGauntlet.Core.ViewModels

open System
open RetroGameGauntlet.Core.Models

/// The Wikipedia link view model.
type WikipediaItemViewModel(model: WikipediaPageModel) = 
    member this.Title = model.Title
    member this.Description = model.Description
    member this.Url = "https://en.wikipedia.org/wiki/" + this.Title
    member this.Uri = Uri(this.Url)